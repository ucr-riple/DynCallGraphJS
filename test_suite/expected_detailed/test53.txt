{
    'global (test53.js:1:1:22:5)': [
        'anon (test53.js:1:2:22:2)'
    ],
    'anon (test53.js:10:17:10:28)': [
        'name (test53.js:3:3:5:4)'
    ],
    'name (test53.js:4:5:4:12)': [
        'anon (test53.js:6:12:8:4)'
    ],
    'anon (test53.js:10:5:10:29)': [
        'extend (test53.js:12:3:21:4)'
    ]
}