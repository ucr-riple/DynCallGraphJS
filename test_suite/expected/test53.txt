{ '(test53.js:1:1:22:5)':
   [ '(test53.js:1:2:22:2)' ],
  '(test53.js:10:17:10:28)':
   [ '(test53.js:3:3:5:4)' ],
  '(test53.js:4:5:4:12)':
   [ '(test53.js:6:12:8:4)' ],
  '(test53.js:10:5:10:29)':
   [ '(test53.js:12:3:21:4)' ] }